.github/workflows/crawler.yml

```diff
@@ -137,7 +137,7 @@ jobs:
       PR_NAME: ${{ github.event.pull_request.title }}
       PR_AUTHOR: ${{ github.actor }}
     runs-on: ubuntu-latest
-    timeout-minutes: 4
+    timeout-minutes: 10
```

Comment:
discard

Reply:
CI takes so long just because you do not control number of collected URLs correctly